string keyword
---
uint8  error_code
